test/ModAttachr/modAttachr.ts

```diff
@@ -0,0 +1,36 @@
+import { ModAttachr } from "../../src/ModAttachr";
+
+import { mochaLoader } from "../main";
+
+import { EventNames } from "../../src/EventNames";
+
+import {
+    IMod
+} from "../../src/IModAttachr";
+
+mochaLoader.it("Creates a mod and fires the contained events", (): void => {
```

Comment:
This is testing that mod events are fired. Creating a mod is part of that, but not the purpose.